tests/atomizer.js

```diff
@@ -42,6 +42,12 @@ describe('Atomizer()', function () {
             var expected = ['Pos(r)', 'Ov(h)', 'H(0)'];
             expect(result).to.deep.equal(expected);
         });
+        it('returns an array of valid atomic class names if the boundary character before the first found classname is a closing brace (Dust.js)', function () {
```

Comment:
This seems to be a cut and paste error - the notion of "first found classname" doesn't apply here (in another test, that refers to a classname that's the first thing found in a file).  I'd change this to something like:

"properly finds Atomic classnames inside Dust template conditionals"

Or if there are other use cases besides Dust templates that could result in curly braces as boundary chars, you could adjust accordingly.